docs/exploration-and-organization/events-and-timelines.md

```diff
@@ -34,39 +36,86 @@ When viewing a [collection](collections.md), you can view, add, or edit events b
-- the timeline is visible (more on that [below](#adding-events-when-viewing-a-question))
+- The date of the event falls within the chart's time range, and
+- The timeline is visible (more on that [below](#adding-events-when-viewing-a-question))
 
 You'll see an icon along the x-axis that plots the event. A vertical line will extend from the event to show when the data plotted on the chart intersects with the event.
 
 ![An event on a chart](./images/example-event.png)
 
-### Adding events when viewing a question
+## Adding events when viewing a question
 
 If your question is a time series, you can click on the **Calendar** in the bottom right of the question, and Metabase will open the timeline sidebar. Metabase will list any timelines and their events that fall in the range of your time series. You can:
 
 - Toggle timeline visibility (including timelines from other collections)
 - Add a new event (even if you haven't saved the question yet).
 - Edit, move, or archive an event.
 
-## Archiving and deleting
+## Viewing events and timelines on a chart from a different collection
+
+If you're viewing a question with a time series chart from a _different_ collection, you can temporarily apply a timeline to the chart by clicking on the **calendar** icon in the bottom right of the question and selecting the timeline and events you want to display.
+
+These selections are temporary; if you reload the question, Metabase will drop the events from the chart.
+
+### To hide the timeline and its events on a chart
+
+To temporarily hide the events from a chart:
+
+1. Click on the **calendar** icon in the bottom right.
+2. Uncheck the timeline or event.
+
+Your selections will reset on reload.
+
+To permanently hide the timeline and its events, you'll need to [archive the timeline](#archiving-timelines). Alternatively, you could move the question to a different collection.
+
+## Edit an event
 
-### Archiving events
+![Edit an event](./images/edit-an-event.png)
 
-You can archive both events and timelines to get them out of the way, or unarchive them to resurrect them.
+To edit an event:
 
-To archive an event, click on the **...** next to the event and select **Archive event**.
+1. Click the calendar icon in the top right of a collection.
+2. Go to the timeline that contains the event, click on the event's three-dot menu (**...**).
+3. Select:
+   - **Edit event**: its title, description, and icon.
+   - **Move event**: to another timeline.
+   - **Archive event**: to hide the event from charts.
+
+## Archiving timelines
+
+To archive a timeline:
+
+1. Go to the timeline's collection and click on the **calendar** icon in the top right.
+2. Select the timeline, then click on the three-dot menu (**...**). If the collection only has one timeline, click on the three-dot menu (**...**).
+3. Select **Edit timeline details**.
+4. Click on the red **Archive timeline and all events**.
+
+### View archived events and timelines
```

Comment:
Maybe worth specifying that archived events and timelines won't show up in what we call "the archive"